### PR TITLE
update docstrings of Accumulator and clean up codes based on its docstrings.

### PR DESCRIPTION
**Pull request by AtsushiSakai**
Update docstrings of Accumulator and clean up codes based on its docstrings.

### CODE REVIEW

**Comment by AtsushiSakai** [comment]
I can change the key "x" to "k".
Do other contributors have any other ideas?

**Comment by oxinabox** [comment]
> not sure about opinion of others contributors... but I personally don't like the use of x for key... maybe this PR could be used to name it k.

I am fine with `x`
accumulators are not exactly dictionaries (I mean technically they are), 
 thinking of it is a key  is a bit funny.

File: src/accumulator.jl [hunk 1]
```diff
@@ -1,5 +1,10 @@
 #A counter type
 
+"""
+    Accumulator{T, V<:Number}
+
+A accumulator is a data structure that maintains an accumulated number for each key.
```
Comment (oxinabox) [inline src/accumulator.jl]:
```suggestion
A accumulator is a data structure that maintains an accumulated total for each key.
The particular case where those totals are integers is a counter.
```

**Comment by AtsushiSakai** [comment]
@oxinabox I fixed it based on your suggestion.

**Comment by oxinabox** [comment]
thanks again for all this.

**Pull request merged**